client/reducers/settings.js

```diff
@@ -20,10 +20,26 @@ export const settings = createReducer(fromJS(initialState), {
       loading: false,
       error: `An error occured while loading the connections: ${action.errorMessage}`
     }),
-  [constants.FETCH_SETTINGS_FULFILLED]: (state, action) =>
-    state.merge({
+  [constants.FETCH_SETTINGS_FULFILLED]: (state, action) => {
+    const data = fromJS(action.payload.data);
+    const settings = data.get('settings');
+    const title = settings.get('title');
+    if(title !== '') document.title = title;
```

Comment:
The title and css logic should not go here. Add it to `client/app.js` by subscribing to the event